src/docs/options.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: 'Options'
+---
+
+Options are a way to store generic system-wide configuration. They serve a similar purpose to configuration files, but they are backed by a database, so it's possible to change them at runtime without a deploy. This makes options well-suited for rates, quotas, and limits.
+
+## Adding New Options
+
+To add a new system option, add it to [`sentry/options/defaults.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/options/defaults.py) by registering it. e.g.,
+
+```python
+register("performance.some-feature-rate", default=0.0)
+```
+
+### Tips
```

Comment:
Should we rephrase this from tips to rules on making an option? I'm thinking if I saw a PR with an option name that wasn't namespaced I would definitely comment on it

Reply:
Done!